app/packages/looker-3d/src/fo3d/Gizmos.tsx

```diff
@@ -20,7 +27,7 @@ const FoAxesHelper = ({
 }) => {
   const { upVector } = useFo3dContext();
   const size = useMemo(
-    () => maxInOrthonormalPlane * 20,
+    () => maxInOrthonormalPlane * 10 + 100,
```

Comment:
Could you make these easier to understand...? I have not idea why you are adding these random numbers here.